CONTRIBUTING.md

```diff
@@ -308,3 +308,14 @@ Check our [documentation writing guide](https://github.com/huggingface/transform
 for more information.
 
 #### This guide was heavily inspired by the awesome [scikit-learn guide to contributing](https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md)
+
+
+### Develop on Windows
+
+The Windows developers will need some more tools to properly run and test transformers. Here a minimum list of the tools that have to be installed:
+
+1. Install [Miniconda for Windows](https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe)
```

Comment:
Why miniconda specifically? There are plenty of other ways to use Python on Windows.

Reply:
There are, but here I give at least one.

Reply:
There are also plenty of other ways to use/install make.

Reply:
I don't see why we would recommend one way to install Python for Windows when we don't for other OS. It's also not in the right place: you can't really install transformers without python. I'd leave the section to "Here is one way you can run the make commands"